AssemblyInfo.Shared.cs

```diff
@@ -26,8 +26,8 @@
 [assembly: AssemblyFileVersion("9.1.0.0")]
 [assembly: AssemblyInformationalVersion("Version:9.1.0.0 Branch:not-set Sha1:not-set")]
 [assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("SonarSource and Microsoft")]
-[assembly: AssemblyCopyright("Copyright © SonarSource and Microsoft 2015-2023")]
+[assembly: AssemblyCompany("SonarSource")]
+[assembly: AssemblyCopyright("Copyright © SonarSource 2015-2023")]
```

Comment:
2023? 🤔

Reply:
It's like this year never been :). I'll set it to 2025 directly